src/NHibernate/Logging.extensions.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+
+namespace NHibernate
+{
+	/// <summary>
+	/// Extensions method for logging.
+	/// </summary>
+	public static class NHibernateLoggerExtensions
+	{
+		public static bool IsDebugEnabled(this INHibernateLogger logger) => logger.IsEnabled(NHibernateLogLevel.Debug);
+		public static bool IsInfoEnabled(this INHibernateLogger logger) => logger.IsEnabled(NHibernateLogLevel.Info);
+		public static bool IsWarnEnabled(this INHibernateLogger logger) => logger.IsEnabled(NHibernateLogLevel.Warn);
+		public static bool IsErrorEnabled(this INHibernateLogger logger) => logger.IsEnabled(NHibernateLogLevel.Error);
+		public static bool IsFatalEnabled(this INHibernateLogger logger) => logger.IsEnabled(NHibernateLogLevel.Fatal);
+
+		public static void Fatal(this INHibernateLogger logger, Exception exception, string format, params object[] args)
+		{
+			logger.Log(NHibernateLogLevel.Fatal, new NHibernateLogValues(format, args), exception);
+		}
+
+		public static void Fatal(this INHibernateLogger logger, Exception exception, string format)
+		{
+			logger.Log(NHibernateLogLevel.Fatal, new NHibernateLogValues(format, null), exception);
+		}
+
+		public static void Fatal(this INHibernateLogger logger, string format, params object[] args)
+		{
+			logger.Log(NHibernateLogLevel.Fatal, new NHibernateLogValues(format, args), null);
+		}
+
+		public static void Fatal(this INHibernateLogger logger, string format)
```

Comment:
A bit strange to have a `format` parameter without argument. Maybe would be better to rename it `message` for this no args overload.

Reply:
Agree